frontend/src/container/Licenses/ApplyLicenseForm.tsx

```diff
@@ -66,16 +71,19 @@ function ApplyLicenseForm({
 
 	return (
 		<ApplyFormContainer>
-			<ApplyForm layout="inline" onFinish={onFinish}>
-				<LicenseInput labelAlign="left" name="key">
-					<Input
-						onChange={(e): void => {
-							setKey(e.target.value as string);
-						}}
-						placeholder={t('placeholder_license_key')}
-					/>
+			<ApplyForm
+				form={form}
+				layout="inline"
+				onFinish={onFinish}
+				autoComplete="off"
+			>
+				<LicenseInput
+					name="key"
+					rules={[{ required: true, message: requireErrorMessage('License Key') }]}
+				>
+					<Input placeholder={t('placeholder_license_key')} />
 				</LicenseInput>
-				<FormItem>
+				<Form.Item>
 					<Button
 						loading={loading}
 						disabled={loading}
```

Comment:
maybe disabling the form when the key length is zero ?

Reply:
Alright

Reply:
Done in #2775